.github/workflows/auto-release.yml

```diff
@@ -0,0 +1,28 @@
+name: Releases
+
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    steps:
+      - name: GitHub App token
+        id: github_app_token
+        uses: tibdex/github-app-token@v1.5.0
+        with:
+          app_id: ${{ secrets.APP_ID }}
```

Comment:
@saratvemulapalli do we need to configure APP_ID & APP_PRIVATE_KEY for this repository individually? thanks!

Reply:
@reta APP_ID and APP_PRIVATE_KEY are already configured for this repo in the repo settings.

Reply:
I was able to test this on my fork of security-dashboards-plugin by creating a new github app and installing it on my fork

Reply:
Thanks @cwperks !